app/Models/OrderDetail.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class OrderDetail extends Model
+{
+    use HasFactory;
+
+    protected $table = 'order_detail';
+    protected $fillable = ['order_id', 'product_id', 'quantity', 'price'];
+    public $timestamp = false;
+
+    public function product()
+    {
+        return $this->hasOne(Product::class, 'id', 'product_id');
```

Comment:
you don't need foreign key when you name the column following the laravel's standard